README.md

```diff
@@ -19,15 +19,15 @@ To plug-in Memcached cache in your application follow the steps below:
    * **Gradle**
 
       ```groovy
-      compile('io.sixhours:memcached-spring-boot-starter:2.3.0')
+      compile('io.sixhours:memcached-spring-boot-starter:2.3.1')
```

Comment:
Can you change from `compile` to `implementation`

Reply:
Good catch. I'll change this.